client/me/purchases/payment/edit-payment-method/credit-card.jsx

```diff
@@ -1,145 +0,0 @@
-/**
- * External Dependencies
- */
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
-import React from 'react';
-
-/**
- * Internal Dependencies
- */
-import Button from 'components/button';
-import classNames from 'classnames';
-import Dialog from 'components/dialog';
-import FoldableCard from 'components/foldable-card';
-import { getPurchase, goToEditCardDetails } from 'me/purchases/utils';
```

Comment:
I suspect we can now remove the `goToEditCardDetails()` function since it doesn't appear to be used anywhere else.